code/modules/clothing/head/collectable.dm

```diff
@@ -14,6 +14,7 @@
 	name = "collectable slime cap!"
 	desc = "It just latches right in place!"
 	icon_state = "slime"
+	dynamic_hair_suffix = "null"

```

Comment:
Don't do this, do ```dynamic_hair_suffix = ""```
At the moment you're loading the image called "null", which doesn't exist, <i>so it just happens to 'work', as in, not do anything as you'd expect</i> however in situations like this you should use true null ```null```, but since it's a string, ```""``` instead, so that the dynamic hair code knows to instead fallback to the object's HIDE_HAIR flag (if applicable)

tl;dr ```dynamic_hair_suffix  = "null"``` -> ```dynamic_hair_suffix = ""```